Tribes/src/app/mock-building.ts

```diff
@@ -0,0 +1,10 @@
+import { BuildingService } from './building.service';
+import { Building } from './building';
+
+
+export const BUILDINGS: Building[] = [
+    new Building (1, 'townhall', 1, 1, 2),
+    new Building (2, 'barraks', 1, 1, 2),
+    new Building (3, 'mine', 1, 1, 2),
+    new Building (4, 'farm', 1, 1, 2)
```

Comment:
if I add a new farm here, then in the `kingdom` page will be a 2 farm section, so maybe we need some sort of filter in the service, so we get just one of each building type to display, or some sort of starting data, for the `kingdom` page

Reply:
Done